<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhysicalDrivesDialog</class>
 <widget class="QDialog" name="PhysicalDrivesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Open Drive</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>451</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="openButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>200</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QLabel" name="adminWarningLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>10</pointsize>
     <italic>true</italic>
     <bold>true</bold>
     <underline>true</underline>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>To use this feature run this application as Administrator</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
